.checkov.baseline

```diff
@@ -27,7 +27,8 @@
                 {
                     "resource": "module.extensions-staging.module.alb.aws_security_group.alb-security-group",
                     "check_ids": [
-                        "CKV_AWS_23"
+                        "CKV_AWS_23",
+                        "CKV2_AWS_5"
```

Comment:
Are these the things @ejayesh has fixed in his pull request?

Reply:
This is what Jayesh asked me to add as the fix for Checkov

Reply:
You know what I think about that!

Reply:
I do. Maybe a ticket to rewrite the security groups and alb resources in the alb module is needed to address that.

Reply:
I will accept this and check/follow up after @ejayesh's merge.